_posts/2024-08-27-cheri-myths-safe-languages.markdown

```diff
@@ -0,0 +1,156 @@
+---
+layout: post
+title:  "CHERI Myths: I don't need CHERI if I have safe languages"
```

Comment:
Pedantic: this is only one myth. Plural is ok for category but doesn't work for the title. Same for writing cheri c post.